apps/examples/src/examples/Svg.tsx

```diff
@@ -131,14 +139,27 @@ class SvgNativeMethods extends Component {
   state = {
     base64: null,
   };
+
+  size = {width: 150, height: 100};
+
+  optionsWithJPEGFormat: DataUrlOptions = {
+    format: 'jpeg',
+    quality: 1,
+    size: {width: 150, height: 100},
+  };
+
+  optionsWithPNGFormat: DataUrlOptions = {
+    format: 'png',
+    size: {width: 150, height: 100},
+  };
+
```

Comment:
unused values?

Reply:
I wanted to leave that as an example, or is it better to remove that?

Reply:
It does not show anything when it's not used. Let's remove that for now and just add a simple inline object to `toDataUri` function execution below.